doc/TYPES2021/acknowledgments.tex

```diff
@@ -1,13 +1,14 @@
 \paragraph*{Acknowledgments}
-This work would not have been possible without the wonderful \agda language and the
-\agdastdlib, developed and
-maintained by The Agda Team~\cite{agdastdlib}.
+This work would not have been possible without the wonderful \agda language and
+the \agdastdlib, developed and maintained by The Agda Team~\cite{agdastdlib}.
 % (Andreas Abel, Guillaume Allais, Liang-Ting Chen, Jesper Cockx, Nils Anders Danielsson,
 % Víctor López Juan, Ulf Norell, Andrés Sicard-Ramírez,Andrea Vezzosi, and Tesla Ice Zhang)
 % , maintained by Matthew Daggitt and Guillaume Allais.
-Most of the content of this paper was generated from the literate \agda file \HSPlagda and the \LaTeXe file \agdahsp (processed with the commands
-\texttt{agda --latex} and \texttt{pdflatex}, respectively), which are available in the \agdaalgebras GitHub repository~\cite{ualib_v2.0.1}.
-The first author was supported by the CoCoSym Project under the ERC Consolidator Grant (ERC CoG), No. 771005.
+%Most of the content of this paper was generated from the literate \agda file
```

Comment:
Hmmm... I'm curious as to why you'd rather not mention the tools we use to generate the manuscript.  I think the people who developed those tools would be very happy to see them mentioned.

Reply:
That kind of information belongs with the software itself, not in the paper.

Reply:
Fair enough.